src/api/packers/Packer.ts

```
@@ -0,0 +1,54 @@
+	packerMethod: string,
+	allowRotation: boolean
+}>;
+
+const METHODS = {
+	Default: "Default"
+} as const;
+
+class Packer {
+	constructor(width: number, height: number, allowRotate: boolean = false, padding: number = 0) {
+		// nothing to do
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)